### PR TITLE
Introductory content for templates

### PR DESCRIPTION
**Pull request by sanderson**
Adds introductory content to landing page for Influx Templates as well as outlines for supporting pages.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/influx-templates/_index.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ---
 title: Influx Templates
 description: >
-  placeholder
+  Influx Templates are prepackaged InfluxDB configurations that contain
```
Comment (kelseiv) [inline content/v2.0/influx-templates/_index.md]: InfluxDB templates are prep....

File: content/v2.0/influx-templates/_index.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+  everything from dashboards and Telegraf configurations to notifications and
+  alerts in a single manifest file.
```
Comment (kelseiv) [inline content/v2.0/influx-templates/_index.md]: ...alerts.

File: content/v2.0/influx-templates/_index.md [hunk 3]
```diff
@@ -12,4 +14,32 @@ from dashboards and Telegraf configurations to notifications and alerts in a sin
 Use Influx Templates to quickly set up a fresh instance of InfluxDB, backup your
```
Comment (kelseiv) [inline content/v2.0/influx-templates/_index.md]: Use InfluxDB templates.....back up....

File: content/v2.0/influx-templates/_index.md [hunk 4]
```diff
@@ -12,4 +14,32 @@ from dashboards and Telegraf configurations to notifications and alerts in a sin
...
 dashboard configuration, or share your configuration with the InfluxData community.
 
+## Template manifests
+A template manifest is a file that defines the contents of a template.
```
Comment (kelseiv) [inline content/v2.0/influx-templates/_index.md]: ## Templates
An InfluxDB template defines....and supports the following formats:

File: content/v2.0/influx-templates/_index.md [hunk 5]
```diff
@@ -12,4 +14,32 @@ from dashboards and Telegraf configurations to notifications and alerts in a sin
...
+- JSON
+- Jsonnet
+
+_See [Create an Influx Template](/v2.0/influx-templates/create/) for information about
+generating manifest files._
+
+### Template resources
+Template **resources** are elements of the InfluxDB user interface (UI) that can
+be included in a template.
+The following template resources are available:
```
Comment (kelseiv) [inline content/v2.0/influx-templates/_index.md]: ## Template resources/elements
Add the following InfluxDB elements/resources (is one preferable?) to your template:

File: content/v2.0/influx-templates/_index.md (hunk 1 above)
Comment (sanderson) [reply content/v2.0/influx-templates/_index.md]: They are calling them "Influx templates", not "InfluxDB templates." I was treating it as a proper noun in this draft, but I've updated them all.

File: content/v2.0/influx-templates/_index.md (hunk 4 above)
Comment (sanderson) [reply content/v2.0/influx-templates/_index.md]: I think the language around "manifests" is important. Our community and marketing teams are using it all their communications about templates.

Comment (kelseiv) [reply content/v2.0/influx-templates/_index.md]: Curious what value the term adds? Is it something other than a template?

File: content/v2.0/influx-templates/_index.md (hunk 1 above)
Comment (kelseiv) [reply content/v2.0/influx-templates/_index.md]: in the UI? @russorat, is this something we can change? ..."templates" or "InfluxDB templates" or "InfluxData templates"?

**Pull request merged**